suite/tests/linux/clone.c

```diff
@@ -287,7 +294,17 @@ create_thread_clone3(void (*fcn)(void), void **stack, bool share_sighand)
     cl_args.exit_signal = SIGCHLD;
     cl_args.stack = (ptr_uint_t)my_stack;
     cl_args.stack_size = THREAD_STACK_SIZE;
-    int ret = make_clone3_syscall(&cl_args, fcn);
+    int ret = make_clone3_syscall(NULL, sizeof(struct clone_args), fcn);
+    assert(errno == EFAULT);
+
+    ret = make_clone3_syscall((void *)0x123 /* bogus address */,
+                              sizeof(struct clone_args), fcn);
+    assert(errno == EFAULT);
+
+    ret = make_clone3_syscall(&cl_args, CLONE_ARGS_SIZE_MIN_POSSIBLE - 1, fcn);
+    assert(errno == EINVAL);
+
+    ret = make_clone3_syscall(&cl_args, sizeof(struct clone_args), fcn);
```

Comment:
Seems good to have a test of fork (i.e. w/o CLONE_VM) using clone3 too.

Reply:
Done.

Reply:
I don't think there was a test without CLONE_VM for clone either. At least not in Linux.clone. Added that too.